.ci/publish.yml

```
@@ -24,6 +24,7 @@ extends:
   parameters:
     publishExtension: ${{ parameters.publishExtension }}
     vscePackageArgs: --no-dependencies
+    apiScanExcludes: '**/w32appcontainertokens-xtra77na.node'
```

This name changes based on whatever the bundle decides to use

```suggestion
    apiScanExcludes: '**/w32appcontainertokens-*.node'
```